app/src/components/App/index.tsx

```diff
@@ -32,9 +27,9 @@ class App extends React.Component<AppProps> {
         <Layout style={{ background: '#fff' }}>
           <Header className={styles.header}>
             <div className={styles.logo}>
-              <Link to="https://sefglobal.org/">
+              <a href="https://sefglobal.org/">
```

Comment:
This should be work with local setups as well. If we redirect to sefglobal.org it's hard to work on a local environment.

This should be 
`<Link to="/academix">`